aidants_connect_web/views/FC_as_FS.py

```diff
@@ -94,12 +96,36 @@ def fc_callback(request):
     fc_access_token = content.get("access_token")
 
     fc_id_token = content.get("id_token")
-    decoded_token = jwt.decode(
-        fc_id_token,
-        settings.FC_AS_FS_SECRET,
-        audience=settings.FC_AS_FS_ID,
-        algorithm="HS256",
-    )
+
+    try:
+        decoded_token = jwt.decode(
+            fc_id_token,
+            settings.FC_AS_FS_SECRET,
+            audience=settings.FC_AS_FS_ID,
+            algorithm="HS256",
+        )
+    except ExpiredSignatureError:
+        # Due to a change in behavior in FC, this hack does not validate the
```

Comment:
I added this because the test failed consistently because the FC token had expired 10 seconds ago